### PR TITLE
fix(assets/issues-notes.css): prevent word break in issue label

### PR DESCRIPTION
**Pull request by ridhishjain**
It fixes issue #2606 
On changing the window size, the issue label won't break into multiple lines. It will be started from a new line

### CODE REVIEW

**Comment by ridhishjain** [comment]
@sidvishnoi review please

**Comment by sidvishnoi** [comment]
`word-break: keep-all;` would've given same results. Would that be more understandable?

**Comment by marcoscaceres** [comment]
> word-break: keep-all; would've given same results. Would that be more understandable?

That seems better, I think. Semantically at least.

**Comment by marcoscaceres** [comment]
@ridhishjain, when you can, could you update the PR with @sidvishnoi's suggestion? Thanks in advance!

**Comment by ridhishjain** [comment]
@marcoscaceres @sidvishnoi the `word-break: keep-all;` does not seem to fix the line break.
I think `white-space: nowrap;` is the best solution for this
Screenshot (52)_LI

**Comment by sidvishnoi** [comment]
Works in Firefox (Webcompat bug?)
image

Works in both Firefox and Chrome with `display: inline-block;`

**Comment by marcoscaceres** [comment]
@sidvishnoi... yeah, retracting my approved. I have to agree that `inline-block` feels more right here.

File: assets/issues-notes.css [hunk 1]
```diff
@@ -50,6 +50,7 @@ input.task-list-item-checkbox {
   position: relative;
   bottom: 2px;
   border: none;
+  white-space: nowrap;
```
Comment (marcoscaceres) [inline assets/issues-notes.css]:
```suggestion
  display: inline-block;
```

**Comment by ridhishjain** [comment]
> I think it was really useful to explore the different alternatives.

I can't agree more ;)

**Comment by sidvishnoi** [comment]
Congrats @ridhishjain for your getting your first PR merged in ReSpec! :tada:
Let me know if you'd like another one

**Comment by ridhishjain** [comment]
> Congrats @ridhishjain for getting your first PR merged in ReSpec! 🎉
> Let me know if you'd like another one

@sidvishnoi Of course, I would love to contribute. Just will require help sometimes as I am still new to the code 😅

**Pull request merged**